pkg/scalers/datadog_scaler.go

```diff
@@ -127,6 +127,9 @@ func parseDatadogMetadata(config *ScalerConfig) (*datadogMetadata, error) {
 
 	meta.datadogSite = siteVal
 
+	if !strings.Contains(meta.query, "{") {
```

Comment:
looking good, just a minor nit: could you please move this up a little bit, next to the lines where the `query` parameter is being resolved, so the query validation related stuff is together. Around lines 66